src/components/k8s/cni/polycube/polycube.go

```diff
@@ -307,9 +316,69 @@ func cmdAdd(args *skel.CmdArgs) error {
 	}
 
 	success = true
+
+	//	If everything's all right, then create the firewall
+	if err := createFirewall(portName, ip.String()); err != nil {
+		log.Errorln("Could not create firewall.")
+		return err
+	}
+
 	return types.PrintResult(result, cniVersion)
 }
 
+func createFirewall(portName, ip string) error {
```

Comment:
Unfortunately not. Just tested:
1) doing so in `CreateFirewallByID` throws 500 if including the chains. Instead, excluding the chains takes no effect for `Interactive: false` as this is totally ignored, so I have to do it on a separate request.
2) setting the chains to `forward` together in `CreateFirewallChainListByID` or `UpdateFirewallChainListByID` (instead of two separate requests) both throw 500.

Reply:
Ok, this looks like a problem on the firewall.

Reply:
@mbertrone Matteo, do you have any idea for this?

Reply:
btw, if I recall correctly the `Interactive: false` being ignored is most probably a well-known issue of the swagger-generated API not sending it in the request (the same happens with 0 and ""). We already discussed this some time prior and it was already there.

Reply:
UPDATE: I can confirm that `Interactive: false` is ignored by the swagger API, not by the Polycube. In issue 184 (https://github.com/polycube-network/polycube/issues/184) there is an example that confirms this.